lib/brasil_api/ncm.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module BrasilAPI
+  class NCM < Base
+    class << self
+
+      def search_ncm(code=nil)
+        get("/ncm/v1", {:search => code})
+      end
+
```

Comment:
Remover linhas em branco
```suggestion
      def search_ncm(code=nil)
        get("/ncm/v1", {:search => code})
      end
```